### DISCUSSION TITLE
specified maven 3.2.5 install throw error with  Unsupported special dependency :java

### DISCUSSION THREAD

**Discussion by Assassin199108** [Homebrew/discussions#5619] [Everyday usage] [answered]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.3.23-25-ge1ff668
ORIGIN: https://github.com/Homebrew/brew
HEAD: e1ff668957dd8a66304c0290dfa66083e6c7444e
Last commit: 8 hours ago
```

### Output of `brew doctor`

```shell
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some taps are not on the default git origin branch and may not receive
updates. If this is a surprise to you, check out the default branch with:
  git -C $(brew --repo wangwei/maven-3.2.5) checkout
  git -C $(brew --repo wangwei/maven-3.5.4) checkout
```

### Description of issue

brew tap-new $USER/maven-3.2.5
brew extract --version=3.2.5 maven $USER/maven-3.2.5
brew install $USER/maven-3.2.5/maven@3.2.5
and then  failed

Error: $USER/maven-3.2.5/maven@3.2.5: Unsupported special dependency :java

**Comment by daeho-ro** [comment]
modify :java to "openjdk" that might works.

**Reply by Assassin199108** [reply]
how to modify openjdk. 
i modify maven@3.2.5.rb line 7 to  [depends_on :openjdk]
but it do not work. still throw error maven-3.2.5/maven@3.2.5: Unsupported special dependency :openjdk

**Reply by daeho-ro** [reply, marked as answer]
I mean `depends_on :java` into `depends_on "openjdk"`.

**Reply by Assassin199108** [reply]
it works! thks very much！
